gui/build.py

```diff
@@ -79,25 +81,20 @@ def do_bundle(options):
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
-    # create a intaller
-    python_lib_path = os.path.join(options.python_path, "Lib")
-    makensis_path = os.path.join(options.nsis_path, "makensis.exe")
+    # create a installer
     gui_path = os.path.realpath('.')
     moz_path = os.path.realpath('..')
-    python_dir = os.path.join(
-        os.path.dirname(sys.executable),
-        "Scripts"
-    )
-    args = []
-    args.append('--icon=wininst/app_icon.ico')
-    args.append('--base-name=Win32GUI')
-    args.append('--include-path=%s;%s;%s\site-packages;%s'
-                % (gui_path, moz_path, python_lib_path, python_lib_path))
-    args.append('--target-name=mozregression-gui.exe')
-    args.append('--target-dir=dist')
-    args.append('mozregui/main.py')
-    call(sys.executable, os.path.join(python_dir, 'cxfreeze'), *args)
-    call(makensis_path, 'wininst.nsi', cwd='wininst')
+    if IS_WIN:
+        python_lib_path = os.path.join(options.python_path, "Lib")
+        makensis_path = os.path.join(options.nsis_path, "makensis.exe")
+        call_cx_freeze(gui_path, moz_path, python_lib_path)
```

Comment:
Once the recommendation above will be done, this call_cx_freeze could be executed for both win and Linux - without testing for the os (outside the "if").